### PR TITLE
Sigil koans

### LINKED ISSUE #60: Koans about sigils
http://elixir-lang.org/getting-started/sigils.html

### PR DESCRIPTION
**Pull request by iamvery**
Ported over from https://github.com/iamvery/elixir-koans/blob/7705d743acf3720f75852eb6badfe57f428a3481/about_sigils.exs

[Closes #60]

### CODE REVIEW

File: lib/koans/03_sigils.ex [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  koan "The lowercase ~s sigil supports string interpolation" do
+    assert ~s[1 + 1 = #{1+1}] == ___
+  end
+
+  koan "The ~S sigil is similar to ~s but doesn't do interpolation" do
+    assert ~S[1 + 1 = #{1+1}] == ___
+  end
+
+  koan "The ~w sigil creates word lists" do
+    assert ~w(Hello world) == [___, ___]
```
Comment (felipesere) [inline lib/koans/03_sigils.ex]: Isn't the fact that a list is being created obvious enough from the title?
I have gut feeling that this is 'too easy' 😅

**Comment by felipesere** [comment]
LGTM 👍

File: lib/koans/03_sigils.ex (hunk 1 above)
Comment (iamvery) [inline lib/koans/03_sigils.ex]: Haha, probably so 😊

Comment (iamvery) [inline lib/koans/03_sigils.ex]: Updated for _MAXIMUM CHALLENGE_ ⚡ 

:joy:

**Pull request merged**